docs/api/effector/Store.md

```diff
@@ -385,21 +385,24 @@ $store.off(trigger)
 ```js
 import {createEvent, createStore} from 'effector'
 
-const changedA = createEvent()
-const changedB = createEvent()
-
+const eventA = createEvent()
+const eventB = createEvent()
 const $store = createStore(0)
 
+$store.on([eventA, eventB], (state, params) => state + params)
 
-// If you want to unsubscribe from all triggers simultaneously, better to manually merge
-const changed = merge([changedA, changedB])
+$store.watch(value => {
+  console.log('updated', value)
+})
 
-$store.on(changed, (state, params) => state + params)
+eventA(2)
+// => updated 2
 
-$store.off(changed)
```

Comment:
You can't remove `.off` from article about `.off`. Also return back `const changed = merge` with comment above it, please